.circleci/config.yml

```diff
@@ -10,6 +10,8 @@ gems-key: &gems-key
   v1-gems-{{ checksum "Gemfile.lock" }}
 brew-key: &brew-key
   v1-brew-{{ checksum "BrewFile" }}
+nightly-key: &nightly-key
+  v0.1-nightly-{{ checksum "~/.pride-nightly/last-commit" }}
```

Comment:
I don't think it's a problem but I think two caches would be generated, one for docker and one for OSX because ~/ expands to a different path in each.